data/text/368.txt

```diff
@@ -1268,3 +1268,4 @@ Playback canceled.
 The dimensions became distorted!
 The Distortion World’s {STRVAR_1 1, 0, 0}\nappeared!\r
 The {STRVAR_1 8, 1, 0} strengthened\n{STRVAR_1 1, 0, 0}’s power!
+The effects of weather disappeared.
```

Comment:
I'd suggest we model this like Intimidate. It's a little cumbersome, but without the ability popups, we should strive for as much clarity in the ability announcement message as possible.

```suggestion
{STRVAR_1 1, 0, 0}’s {STRVAR_1 5, 1, 0}’s\nsuppressed the effects of weather!
```

You'll also want to add extra strings for a wild Pokemon/enemy battler like so (in order right after the above):

```
The wild {STRVAR_1 1, 0, 0}’s {STRVAR_1 5, 1, 0}’s\nsuppressed the effects of weather!
The foe’s {STRVAR_1 1, 0, 0}’s {STRVAR_1 5, 1, 0}’s\nsuppressed the effects of weather!
```